src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.IntegrationTests/Microsoft.Azure.Services.AppAuthentication.IntegrationTests.csproj

```diff
@@ -0,0 +1,69 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  
+	<PropertyGroup>
+    
+		<TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+  
+		<SignAssembly>True</SignAssembly>
+    	
+		<AssemblyOriginatorKeyFile>$(LibraryToolsFolder)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    
+		<DelaySign>True</DelaySign>
+  
+	</PropertyGroup>
+
+  
+	
+	<ItemGroup>
+    
+		<PackageReference Include="Microsoft.Azure.KeyVault" Version="2.3.2" />
+    
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    
+		<PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    
+		<PackageReference Include="xunit" Version="2.2.0" />
+    
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+  
+	</ItemGroup>
+
+
+	
+<ItemGroup>
+    
+		<ProjectReference Include="..\Azure.Services.AppAuthentication.TestCommon\Microsoft.Azure.Services.AppAuthentication.TestCommon.csproj" />
+    
+		<ProjectReference Include="..\Azure.Services.AppAuthentication\Microsoft.Azure.Services.AppAuthentication.csproj" />
+  
+	</ItemGroup>
+
+  
+
+	<PropertyGroup Condition=" '$(TargetFramework)'=='net452' ">
```

Comment:
the compiler constant as well as property group is part of common targets/props. Please remove